Dockerfile

```diff
@@ -28,16 +28,7 @@ FROM gobase AS docker
 ARG TARGETPLATFORM
 ARG DOCKER_VERSION
 WORKDIR /opt/docker
-RUN DOCKER_ARCH=$(case ${TARGETPLATFORM:-linux/amd64} in \
-    "linux/amd64")   echo "x86_64"  ;; \
-    "linux/arm/v6")  echo "armel"   ;; \
```

Comment:
Actually this doesn't look correct for arm/vX . https://github.com/tonistiigi/xx/blob/master/base/xx-info#L211 . I'm not sure if this is based on anything? Arm is like debian but aarch64/ppc64le are not debian 😕

Reply:
Yes indeed, I recall we had a discussion with @tianon and @thaJeztah to align them in the future.

Reply:
In that case, we could still do a `case` in `xx-info` with fewer checks than the original, or if you think it's not worth it then we can close this and keep as is.

Reply:
Yes let's keep it as is. I'm closing this one and mark it as needs/follow-up.